apps/asctb-reporter/src/app/components/compare/compare.component.ts

```diff
@@ -52,24 +52,40 @@ import { SidenavModule } from '../sidenav/sidenav.module';
   styleUrl: './compare.component.scss',
 })
 export class CompareComponent implements OnInit {
-  readonly fb = inject(UntypedFormBuilder);
+  /** Observable of compare data sheets */
+  readonly compareSheets = input.required<Observable<CompareData[]>>();
```

Comment:
Why does this take an observable? It makes more sense to refactor it to take an array and when passing the data in do `[compareSheets]="(source | async) ?? []"`